tools/clang/include/clang/SPIRV/SpirvContext.h

```diff
@@ -483,6 +487,8 @@ class SpirvContext {
   uint32_t majorVersion;
   uint32_t minorVersion;
 
+  bool signaturePacking; ///< Whether signature packing is enabled or not
```

Comment:
I think we should make this its own `SpirvCodeGenOptions` and set and access it like any other SPIR-V option. I don't think we handle any other options that are this granular by setting a member variable on `SpirvContext`.

Reply:
done